inventory/all_projects/postgresql

```diff
@@ -14,7 +14,9 @@ figgy-db-staging2.princeton.edu
 postgres-staging1.lib.princeton.edu
 lib-postgres-staging1.princeton.edu
 lib-postgres-staging2.princeton.edu
-lib-postgres-staging3.princeton.edu
+[lib-postgres_staging_cluster]
```

Comment:
Suggestion: Remove this group, I think you ended up using `postgres_staging_cluster`